src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -375,6 +377,9 @@ class ContentSidebar extends React.Component<Props, State> {
         }: Props = this.props;
         const { file, isLoading, metadataEditors }: State = this.state;
         const initialPath = defaultView.charAt(0) === '/' ? defaultView : `/${defaultView}`;
+        
+        // Filter out falsy values from customSidebarPanels array
+        const filteredCustomSidebarPanels = customSidebarPanels ? customSidebarPanels.filter(Boolean) : undefined;
```

Comment:
why would there be falsy values?

Reply:
SInce `boxai.sidebar.enabled` aka `hasBoxAI` will not longer be passed in, In EUA I will use something like `boxai.sidebar.enabled && customSidebarPanel`, which leads to `falsy` values when `boxai.sidebar.enabled` is `false`